### ISSUE TITLE
NullReferenceException in Bittrex.Net.Clients.BittrexSocketClient.Check

### ISSUE THREAD

**Issue by EricGarnier** [JKorf/Bittrex.Net#207]
Hi @JKorf ,
I try our latest version of Bittrex.Net and got an exception on the order update subscription.

Here the trace log (with some parts replace by XXX, and some cuts)
[Extract.txt](https://github.com/JKorf/Bittrex.Net/files/8108353/Extract.txt)

```
2022/02/21 10:00:49:635 | Error | Bittrex    | Socket 3 Exception during message processing
Exception: NullReferenceException - Object reference not set to an instance of an object.
   at Bittrex.Net.Clients.BittrexSocketClient.Check(String channel, String method, String data)
   at Bittrex.Net.Clients.BittrexSocketClient.MessageMatchesHandler(SocketConnection socketConnection, JToken message, Object request)
   at CryptoExchange.Net.Sockets.SocketConnection.HandleData(MessageEvent messageEvent)
```
Do you have an idea what goes wrong?
What can I do to help ?

Thanks
Eric.

**Comment by JKorf** [comment]
Hi, thanks for the report. This should be fixed in the latest version 7.01. Let me know if you have any other issues

**Comment by EricGarnier** [comment]
Hi, seems you fixed it just right. You nailed it down ;-)

Thanks.
